src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -69,7 +74,7 @@ function KeylessBackupProgress({
   if (route.params.keylessBackupFlow === KeylessBackupFlow.Restore) {
     return <Restore />
   } else {
-    return <Setup />
+    return <Setup origin={route.params.origin} />
```

Comment:
nit: this can also be destructured with an additional `const { origin } = route.params` in the code above.